README.md

```diff
@@ -1,2 +1,16 @@
 # ramble
-Ramble 
+[![Stories in Ready](https://badge.waffle.io/ascasson/ramble.png?label=ready&title=Ready)](http://waffle.io/ascasson/ramble)
```

Comment:
Its great that yall included a readme, but this could have included a project description, contriubting gudid, and link to the backend app :)